troubleshooting/known_issues/hpe_systems_not_transitioning_power_state.md

```diff
@@ -0,0 +1,33 @@
+# HPE iLO dropping event subscriptions and not properly transitioning power state in CSM software
+
+HPE Systems impacted:
+
+* DL385
```

Comment:
This really doesn't affect the DL325 nodes?

Reply:
There is evidence that it has happened to the DL325s, but I saw no indication that it was a current issue on any DL325 in-house or in-the-field. I believe our procedures during install resolve potential issue on the DL325.